### PR TITLE
Add mysql and postgresql binaries.

### PR DESCRIPTION
**Pull request by Wnatat**
Adding mysql and postgresql clients for use in dev sync_environment scripts.

### CODE REVIEW

File: awscli/Dockerfile [hunk 1]
```diff
@@ -8,6 +8,8 @@ RUN apk add --update --no-cache \
     curl \
     python3 \
     jq \
+    mysql-client \
+    postgresql \
```
Comment (chinthakagodawita) [inline awscli/Dockerfile]: Is there an equivalent 'client-only' package for postgres?

Comment (chinthakagodawita) [reply awscli/Dockerfile]: @Wnatat take a look at https://github.com/lendingworks/docker-containers/blob/master/php/base/Dockerfile#L31

File: awscli/Dockerfile [hunk 2]
```diff
@@ -8,6 +8,8 @@ RUN apk add --update --no-cache \
     curl \
     python3 \
     jq \
+    mysql-client \
```
Comment (andrewandante) [inline awscli/Dockerfile]: Do we want to lock these to specific versions or let the gods decide?

**Comment by chinthakagodawita** [comment]
Do you need to add `pv` too?

File: awscli/Dockerfile (hunk 2 above)
Comment (Wnatat) [reply awscli/Dockerfile]: I'd think they keep mysql and postgresql clients backward compatible, and that Alpine repo wouldn't necessarily keep all individual version available in the future.

**Comment by Wnatat** [comment]
> Do you need to add `pv` too?
Yes, though it's not having any effect since we use a K8s job for this task.

**Pull request merged**